src/entry.js

```diff
@@ -1,6 +1,20 @@
+import React from 'react'
 import ReactDOM from 'react-dom'
+import { Provider } from 'react-redux'
+
 import routes from './routes'
+import store from './store'
+
 import '../sass/app.scss'
 
-const div = document.getElementById('app')
-ReactDOM.render(routes, div)
+const rootEl = document.getElementById('app')
+const render = () => {
```

Comment:
ok, i'm still not convinced here -- check out this redux / react router example app: https://github.com/reactjs/redux/blob/master/examples/real-world/src/index.js#L11

how are they getting around the `store.subscribe...` line?

i also like this pattern of this `Root` component within `containers` directory